src/main/java/project/dao/jdbc/CarDaoJdbc.java

```diff
@@ -0,0 +1,209 @@
+public class CarDaoJdbc implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String selectQuery = "INSERT INTO cars (car_model, manufacturer_id) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement stInsert = connection.prepareStatement(selectQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            stInsert.setString(1, car.getName());
+            stInsert.setLong(2, car.getManufacturer().getId());
+            stInsert.execute();
+            ResultSet resultSet = stInsert.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject("GENERATED_KEY", Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car " + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectQuery = "SELECT c.car_id, c.car_model,"
+                + "c.manufacturer_id, cd.driver_id FROM cars c "
+                + "INNER JOIN manufacturers m ON m.manufacturer_id=c.manufacturer_id "
+                + "INNER JOIN cars_drivers cd ON cd.car_id=c.id "
+                + "INNER JOIN drivers d ON d.driver_id=cd.driver_id "
+                + "WHERE c.deleted=FALSE AND d.deleted=FALSE AND c.car_id=?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement stSelectById = connection.prepareStatement(selectQuery)) {
+            stSelectById.setLong(1, id);
+            ResultSet resultSet = stSelectById.executeQuery();
+            Car car = null;
+            if (resultSet.next()) {
+                car = getCarFromResultSet(resultSet);
+            }
+            return Optional.of(car);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get car with id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String selectQuery = "SELECT c.car_id, c.car_model,"
+                + "c.manufacturer_id, cd.driver_id FROM cars c "
+                + "INNER JOIN manufacturers m ON m.manufacturer_id=c.manufacturer_id "
+                + "INNER JOIN cars_drivers cd ON cd.car_id=c.car_id "
+                + "INNER JOIN drivers d ON d.driver_id=cd.driver_id "
+                + "WHERE c.deleted=FALSE AND d.deleted=FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement selectAll = connection.prepareStatement(selectQuery)) {
+            ResultSet resultSet = selectAll.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(getCarFromResultSet(resultSet));
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all cars", e);
+        }
```

Comment:
```suggestion
        try (Connection connection = ConnectionUtil.getConnection();
                PreparedStatement selectAll = connection.prepareStatement(selectQuery)) {
            ResultSet resultSet = selectAll.executeQuery();
            List<Car> cars = new ArrayList<>();
            while (resultSet.next()) {
                cars.add(getCarFromResultSet(resultSet));
            }
        } catch (SQLException e) {
            throw new DataProcessingException("Can't get all cars", e);
        }
              for (Car car : cars) {
            car.setDrivers(getDrivers(car.getId()));
        }
        return cars;
```
но в таком случаем тебе придется сделать так, что  метод getDrivers возвращал не просто одного драйвера, а список драйверов, которые относятся к этой машине